src/DynamoPublish/ViewModels/PublishViewModel.cs

```diff
@@ -139,6 +141,10 @@ private void OnPublish(object obj)
             if (!model.IsLoggedIn)
                 return;
 
+            var homeWorkspace = Workspaces.OfType<HomeWorkspaceModel>().First();
+            homeWorkspace.Name = Name;
+            homeWorkspace.Description = Description;
```

Comment:
What if you don't want to change the workspace's name and description when sending?

Instead, let's follow my recommendation detailed here:

http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-8059

This will require passing the name and description as a separate argument object to the API in Reach.dll.